packages/notification/channel/mailgun.ts

```diff
@@ -22,19 +22,27 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
-import Mailgun from 'mailgun.js'
-import formData from 'form-data'
+import FormData from 'form-data'
 import Joi from 'joi'
 import type { NotificationMessage } from '.'
 import Mailgen from 'mailgen'
+import { sendHttpRequest } from '../utils/http'
 
-type NotificationData = {
+export type NotificationData = {
   apiKey: string
   domain: string
   recipients: string[]
   username?: string
```

Comment:
it seems like for mailgun the username must always be `api` https://documentation.mailgun.com/en/latest/api-intro.html#authentication-1. So i think it's better to remove this.